src/Command/GithubReleaseCommand.php

```diff
@@ -0,0 +1,162 @@
+
+    protected static $defaultName = 'github:release';
+
+    /** @var SymfonyStyle */
+    private $io;
+    /** @var HttpClientInterface */
+    private $client;
+
+    private $tag;
+    private $name;
+    private $description;
+
+    public function __construct()
+    {
+        parent::__construct(self::$defaultName);
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Create a release on github with a .phar as attachment.');
+        $this->addArgument('tag', InputArgument::REQUIRED, 'Release\'s tag.');
+        $this->addArgument('name', InputArgument::OPTIONAL, 'Release name', 'Symfony docs builder %s');
+        $this->addArgument('description', InputArgument::OPTIONAL, 'Symfony docs builder %s');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $dotenv = new Dotenv();
+        $dotenv->load(__DIR__.'/../../.env');
+
+        if (empty($_ENV['GITHUB_API_TOKEN'])) {
+            throw new RuntimeException('Please fill "GITHUB_API_TOKEN" in file "[PROJECT_DIR]/.env"');
+        }
+
+        $this->io     = new SymfonyStyle($input, $output);
+        $this->client = $this->createHttpClient($_ENV['GITHUB_API_TOKEN']);
+
+        $tag = $input->getArgument('tag');
+        if (!preg_match('/^v\d+\.\d+\.\d+$/', $tag)) {
+            throw new RuntimeException(sprintf('"%s" is not a valid tag.', $tag));
+        }
+
+        $this->tag         = $tag;
+        $this->name        = sprintf($input->getArgument('name'), $tag);
+        $this->description = sprintf($input->getArgument('description'), $tag);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->compilePhar();
+
+        $this->addAssetToRelease($this->createRelease());
+    }
+
+    /**
+     * @throws ProcessFailedException
+     */
+    private function compilePhar(): void
+    {
+        $process = Process::fromShellCommandline('./bin/compile', __DIR__.'/../..');
+        $process->mustRun();
+    }
+
+    private function createHttpClient(string $githubToken): HttpClientInterface
+    {
+        $client = HttpClient::create(
+            [
+                'headers' => [
+                    'Authorization' => sprintf('token %s', $githubToken),
+                ],
+            ]
+        );
+
+        return $client;
+    }
+
+    private function createRelease(): int
+    {
+        try {
+            $response = $this->client->request(
+                'POST',
+                sprintf('https://api.github.com/repos/%s/%s/releases', self::GITHUB_USER, self::GITHUB_REPO),
+                [
+                    'json' => [
+                        'tag_name'         => $this->tag,
+                        'target_commitish' => 'master',
+                        'name'             => $this->name,
+                        'description'      => $this->description,
+                        'draft'            => false,
```

Comment:
Hmm. Could we set this `draft` => true, then upload the asset, then modify it to `draft=>false`? That's minor... but it would eliminate any time when the release is published and has *no* phar attached.